queries/d/highlights.scm

```diff
@@ -219,6 +200,35 @@
   "try"
 ] @exception
 
+"null" @constant.builtin
+
+[
+  "__gshared"
```

Comment:
"`__gshared` stores the variable in the classic global data segment" (https://dlang.org/articles/migrate-to-shared.html#gshared). I'm not sure if `@type.qualifier` and `@storageclass.lifetime` fits better.